docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm

```diff
@@ -189,11 +191,13 @@ <h3>Summary</h3>
 							prettyprint">
 &lt;input type=&quot;checkbox&quot; name=&quot;checkbox-1&quot; id=&quot;checkbox-1&quot;/&gt;
```

Comment:
According to [this comment](https://github.com/Samsung/tizen-docs/pull/607#discussion_r263327829),
if there are two blocks of code after each other then both descriptions HTML and JS are to be used.

How about these sample code. Don't we need "\<p>HTML code:\</p>" here?

Reply:
Fixed